Source/LinqToDB/DataExtensions.cs

```diff
@@ -78,8 +78,11 @@ public static ITable<T> GetTable<T>(
 
 				for (var i = 0; i < parameters.Length; i++)
 				{
-					var type = pis[i].ParameterType;
-					args.Add(Expression.Constant(parameters[i], (type.IsByRef ? type.GetElementType() : type)!));
+					var        type    = pis[i].ParameterType;
+					Expression argExpr = Expression.Constant(parameters[i], (type.IsByRef ? type.GetElementType() : type)!);
+					argExpr = SequenceHelper.WrapAsParameter(argExpr);
```

Comment:
make `WrapAsParameter` an extension method to simplify this and avoid assigning the variable twice
```suggestion
					Expression argExpr = Expression
						.Constant(parameters[i], (type.IsByRef ? type.GetElementType() : type)!)
						.WrapAsParameter();
```